frontend/src/scenes/session-recordings/playlist/SessionRecordingPreview.tsx

```diff
@@ -153,7 +153,15 @@ function PinnedIndicator(): JSX.Element | null {
     )
 }
 
-function ViewedIndicator(): JSX.Element {
+function RecordingOngoingIndicator(): JSX.Element {
+    return (
+        <Tooltip title="This recording is still ongoing - we received data within the last 5 minutes.">
+            <IconLive />
```

Comment:
This feels like it should be live somewhat:
![pulse](https://github.com/user-attachments/assets/e272cfe3-dc2f-46d2-ae2b-56f9c332b4ee)

You can achieve this with some simple custom Tailwind:
```suggestion
            <IconLive className="animate-[pulse_1s_ease-out_infinite] text-primary-3000" />
```

Reply:
You are now officially better than ChatGPT at making things pulse

Reply:
Test me on my knowledge of the ancient mayan civilisation next... 👀